MOBILE_API.xml

```diff
@@ -2670,11 +2670,20 @@
         </param>
         <param name="rdsData" type="RdsData" mandatory="false">
         </param>
-        <param name="availableHDs" type="Integer" minvalue="1" maxvalue="3" mandatory="false">
+        <param name="hdRadioEnable" type="Boolean" mandatory="false"  since="5.0">
+            <description> True if the hd radio is on, false is the radio is off</description>
```

Comment:
Should this say `false if the radio is off` (instead of `false is the radio is off`)?